City-of-Helsinki/parking-permits
# PV-350: Add parking permit events

## Description

Add parking permit events. Shown in parking permit history in the admin UI. Probably missing a few spots. No events created for temporary vehicle related stuff.

## Context

[PV-350](https://helsinkisolutionoffice.atlassian.net/browse/PV-350)

## How Has This Been Tested?

Some unit tests. Some manual testing. A lot of the code is completely untested.

## Manual Testing Instructions for Reviewers

Create/update parking permit events through the front-end, see if stuff gets generated.
